src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -104,8 +143,24 @@ export default {
             let found = this.findStyleByConditionId(conditionIdentifier);
```

Comment:
Hrm, why does this not just return the conditionalStyle object? Why does it return a new object with the conditional style object AND an index? Sorry, realize it's out of scope for this change, but this looks like a code smell that we should fix.

`findStyleByConditionId` could then be massively simplified to just :
``` JavaScript
findStyleByConditionId(id) {
    return this.conditionalStyles.find(conditionalStyle => 
        this.openmct.objects.areIdsEqual(conditionalStyle.conditionIdentifier, id));
}
```

Reply:
Agreed. Stupid old habits.